config/initializers/content_security_policy.rb

```diff
@@ -1,3 +1,4 @@
+# frozen_string_literal: true
```

Comment:
I see rubocop changed those frozen strings, thats why we had so many files changed!